resources/content/schema/user/user-auth.json

```diff
@@ -43,6 +43,8 @@
 
         "auditLog.resourceActions\\..*" : "",
 
+        "baseMachineConfig": "cr",
```

Comment:
We need to add this so that the dynamic ones that inherit from it will have the proper permisisons? Is it not sufficient to just give the dynamic ones "cr"?